tests/drivers/build_all/sensor/i2c.dtsi

```diff
@@ -1149,6 +1149,12 @@ test_i2c_npm2100: npm2100@9f {
 	};
 };
 
+test_i2c_mlx90394: mlx90394@a0 {
+	compatible = "melexis,mlx90394";
+	status = "okay";
+	reg = <0xa0>;
+};
```

Comment:
These changes should be in a separate commit.

Reply:
See the [documentation](https://docs.zephyrproject.org/latest/contribute/contributor_expectations.html#multiple-commits-on-a-single-pr). Commits should try to keep logical relevant changes to a minimum, adding a feature/driver and adding tests for a feature/driver are two logical changes.